pkg/pillar/cmd/nodeagent/handletimers.go

```diff
@@ -116,7 +116,10 @@ func handleRebootOnVaultLocked(ctxPtr *nodeagentContext) {
 	//This could be due to remote attestation taking time
 	//Check if we have crossed cut off time
 	vaultCutOffTime := ctxPtr.globalConfig.GlobalValueInt(types.VaultReadyCutOffTime)
-	timePassed := ctxPtr.timeTickCount - ctxPtr.vaultTestStartTime
+	var timePassed uint32
+	if ctxPtr.vaultTestStartTime > 0 && ctxPtr.configGetSuccess {
```

Comment:
How about at least comment saying "check vaultTestStartTime to make sure we got a notification from vaultmgr"?

But would be easier to understand if we had a ctxPtr.vaultmgrReported to parallel ctxPtr.configGetSuccess.